doc/background.rst

```diff
@@ -53,3 +59,37 @@ If you are interested to learn more or want to help please :doc:`join our channe
 where we meet next <contact>`.
 
 See :doc:`contents` for an index of all docs and discussion results so far.
+
+
+Design Differences to previous approaches
```

Comment:
Other section headers in this document have consistent capitalization.  Maybe this one should use the same approach?